core/templates/components/oppia-angular-root.component.ts

```diff
@@ -164,6 +164,14 @@ export class OppiaAngularRootComponent implements AfterViewInit {
       const rteElement = createCustomElement(
         componentMap[rteKey].component_class,
         {injector: this.injector});
+      if (
+        customElements.get(
+          'oppia-noninteractive-ckeditor-' +
+          ServicesConstants.RTE_COMPONENT_SPECS[rteKey].frontend_id
+        ) !== undefined
+      ) {
+        continue;
+      }
```

Comment:
When we change from rtl to ltr (and vice versa) I started seeing errors that custom element is already defined. I will check again to make understand why this happens even after an refresh.

Reply:
Now being tracked in a long term issue here: https://github.com/oppia/oppia/issues/16718